kapre/delta.py

```diff
@@ -55,7 +64,6 @@ def call(self, x):
         kernel = K.arange(-n,n+1,1,dtype=K.floatx())
         kernel = K.reshape(kernel,(1,kernel.shape[-1],1,1))
         
-        #output = tf.nn.conv2d(x,kernel,1,'VALID',data_format='NHWC')
         x = K.conv2d(x,kernel,1,data_format='channels_last')/denom
```

Comment:
could you run anything like `autopep8` for the formatting?

Reply:
actually, as in https://github.com/keunwoochoi/kapre/blob/master/tox.ini#L34, you'll need to install `black` and run it 
```
black --line-length 100 --skip-string-normalization --target-version=py37 {toxinidir}/kapre
black --line-length 100 --skip-string-normalization --target-version=py37 {toxinidir}/tests
```
which will do automatic formatting.

Reply:
Ok I will do it

Reply:
Done